x/evm/keeper/config.go

```diff
@@ -17,7 +18,7 @@ import (
 // EVMConfig creates the EVMConfig based on current state
 func (k *Keeper) EVMConfig(ctx sdk.Context, proposerAddress sdk.ConsAddress, chainID *big.Int) (*statedb.EVMConfig, error) {
 	params := k.GetParams(ctx)
-	ethCfg := params.ChainConfig.EthereumConfig(chainID)
+	ethCfg := config.GetChainConfig().EthereumConfig(chainID)
```

Comment:
@ramacarlucho let's remove the `ChainConfig` proto struct and only use the one from geth from the configurator. The conversion from `proto` to the geth type is unnecessary